core/web/resolver/auth.go

```diff
@@ -7,13 +7,13 @@ import (
 )
 
 // Authenticates the user from the session cookie.
-func authenticateUser(ctx context.Context) error {
-	_, ok := auth.GetGQLAuthenticatedUser(ctx)
+func authenticateUser(ctx context.Context) (*auth.GQLQUserSession, error) {
```

Comment:
It would be better to not return the user here and leave it as a pure authentication check.

If you wanted to get the user, you could call `auth. GetGQLAuthenticatedUser` directly in the resolver

Reply:
Understood, I'll do that.